package.json

```diff
@@ -32,6 +32,8 @@
     "typescript": "4.0.2"
   },
   "scripts": {
+    "bench": "cd performance && yarn start",
+    "bench:docker": "docker run -it --rm -v \"$PWD\":/usr/src/bench -w /usr/src/bench -m=\"256m\" --cpus=\"2\" node:14.10 npm run bench",
```

Comment:
Could the executed Docker command be moved to a docker-compose file?

Reply:
What will be the benefits?

Reply:
It can be overridden easily by combining multiple compose files. So a Windows user could override it with a Windows path mapping. Probably no need though unless someone comes asking :)

Reply:
Okay. Sure, it can be done in followups.

Reply:
Other reasons:

- Generally easier to control the flags passed like `-d`, `--verbose` etc - Since `npm run bench` runs after the Docker command here I can't access those Docker flags.